depthmapXcli/commandlineparser.cpp

```diff
@@ -108,6 +109,10 @@ void CommandLineParser::parse(size_t argc, char *argv[])
         {
             m_simpleMode = true;
         }
+        else if ( std::strcmp("-p", argv[i]) == 0)
```

Comment:
Is printing progress not available on simple mode? The `else if` suggests that...

Reply:
The `else if` examines whether a given argument is `-s` or `-p`, so providing both  as different arguments should run simple mode with progress prints